web/pimcore/static6/js/pimcore/asset/listfolder.js

```diff
@@ -148,7 +148,13 @@ pimcore.asset.listfolder = Class.create({
             tbar: [
                 "->"
                 ,this.checkboxOnlyDirectChildren
-                ]
+                , "-"
+                ,{
+                    text: t("download_as_zip"),
```

Comment:
We should use `Download selected items as ZIP` as the label, to differentiate from the `Download as ZIP` button in the toolbar.